_layouts/software-details.html

```diff
@@ -646,11 +646,16 @@ <h2> {{ t.software.interest_related_softwares }} </h2>
         <div class="col-12 relate">
           {% assign relsw_url = '/' | append: active_lang | append: '/software/' | append: relsw.slug | downcase %}
 
+
           <div class="img-container">
             <a href="{{ relsw_url }}">
-              <img class="img-fluid"
-                src="{{ relsw_description.screenshots | first | default: relsw.publiccode.logo | default: '/assets/images/placeholder-software-riuso.jpg' }}"
-                alt="{{ relsw_name }}"></a>
+              {% assign swClasses = "d-inline-block" %}
```

Comment:
This is still not showing up, we need to set some other class or change some CSS..

Reply:
seems a lack of height and width for image

Reply:
Yes exactly however how should we handle it?
Let me know if what I proposed works for you, thanks!